website/public/js/controllers/tasksCtrl.js

```diff
@@ -24,7 +24,10 @@ habitrpg.controller("TasksCtrl", ['$scope', '$rootScope', '$location', 'User','N
               if (direction === 'down') $rootScope.playSound('Minus_Habit');
               else if (direction === 'up') $rootScope.playSound('Plus_Habit');
       }
-      User.user.ops.score({params:{id: task.id, direction:direction}});
+      Tasks.scoreTask(task.id, direction)
```

Comment:
These ops should use the shared code like User.user.ops.score so that we don't have to wait for the server response to update the UI